phys2denoise/metrics/chest_belt.py

```diff
@@ -1,33 +1,38 @@
 """Denoising metrics for chest belt recordings."""
 import numpy as np
 import pandas as pd
+from physutils import io, physio
 from scipy.interpolate import interp1d
 from scipy.stats import zscore
 
 from .. import references
 from ..due import due
 from .responses import rrf
 from .utils import apply_function_in_sliding_window as afsw
-from .utils import convolve_and_rescale, rms_envelope_1d
+from .utils import convolve_and_rescale, return_physio_or_metric, rms_envelope_1d
 
 
 @due.dcite(references.BIRN_2006)
-def respiratory_variance_time(resp, peaks, troughs, samplerate, lags=(0, 4, 8, 12)):
+@return_physio_or_metric()
+@physio.make_operation()
+def respiratory_variance_time(
+    data, fs=None, peaks=None, troughs=None, lags=(0, 4, 8, 12), **kwargs
```

Comment:
It would be good to have peaks and troughs first when possible (and, in general, avoid changing the order of inputs as much as possible), but if it makes things too complicated with the wrapper, then no problem.

Reply:
done